python/paddle/v2/inference.py

```diff
@@ -97,8 +97,22 @@ def infer(output_layer, parameters, input, feeding=None, field='value'):
                               input=SomeData)
         print result
 
+    If there are multiple outout_layers and fields, the examples usages:
+
+    ..  code-block:: python
+
+        result = paddle.infer(output_layer=[prediction1, prediction2], 
+                              parameters=parameters, 
+                              input=SomeData,
+                              field=[id, value]])
+        print result
+
+    The result order is prediction1.id, prediction2.id, prediction1.value, 
+    prediction2.value.
```

Comment:
:return: predicted results. The result order is prediction1.id, prediction2.id, prediction1.value, prediction2.value.
:rtype: list